packages/atomic/src/components/commerce/facets/atomic-commerce-category-facet/atomic-commerce-category-facet.tsx

```diff
@@ -41,6 +44,7 @@ import {FacetShowMoreLess} from '../../../common/facets/facet-show-more-less/fac
 import {FacetValuesGroup} from '../../../common/facets/facet-values-group/facet-values-group';
 import {initializePopover} from '../../../common/facets/popover/popover-type';
 import {CommerceBindings as Bindings} from '../../atomic-commerce-interface/atomic-commerce-interface';
+import css from './atomic-commerce-category-facet.css';
```

Comment:
Why not allCss here?

Reply:
in a `import ns from './path/to/file'`, `ns` can be anything, it's a concern of the importer.
`ns` is just the variable in which `default` from the export will be stored.
I think cssAll for the consumer is noisy, so I elected to call it `css` instead

Reply:
Oh right, forgot about that!